### DISCUSSION TITLE
Landing Page Redesign Concept

### DISCUSSION THREAD

**Discussion by NusryNizam** [friction2d/friction#280] [General]
Hi @rodlie 
I just came up with a concept design for the landing page. 
If this is something you would consider, I'd love to develop this myself.

Feel free to suggest any changes as well.

![Board-2](https://github.com/user-attachments/assets/0796b461-bdce-4a50-920f-9fb7fb8151e1)

![Board](https://github.com/user-attachments/assets/7cd685a7-f0e4-483b-9f86-2d57faac1e7e)

**Comment by thanhhocse96** [comment]
Maybe 1 tab for tutorial

**Reply by NusryNizam** [reply]
Yeah, that would be a useful addition.

**Reply by pgilfernandez** [reply]
And Documentation... Friction needs some info on how to use all its features.
You could probably start by copying and updating the Enve one.

**Reply by luzpaz** [reply]
Tutorials too!
